index.html

```diff
@@ -3657,63 +3657,59 @@ <h3>Parsing content</h3>
-              <ol>
-                <li>
-                  If |context| is not equal to `"smart-poster"`,
-                  [= exception/throw =] a {{TypeError}} and abort these steps.
-                </li>
-                <li>
-                  Set |record| to the result of running
-                  <a>parse a smart-poster size record</a> on |ndef|.
-                </li>
-              </ol>
-              <dt>"`:t`" (`0x53` `0x70`)</dt>
-              <ol>
-                <li>
-                  If |context| is not equal to `"smart-poster"`,
-                  [= exception/throw =] a {{TypeError}} and abort these steps.
-                </li>
-                <li>
-                  Set |record| to the result of running
-                  <a>parse a smart-poster type record</a> on |ndef|.
-                </li>
-              </ol>
-              <dt>"`:act`" (`0x53` `0x70`)</dt>
-              <ol>
-                <li>
-                  If |context| is not equal to `"smart-poster"`,
-                  [= exception/throw =] a {{TypeError}} and abort these steps.
-                </li>
-                <li>
-                  Set |record| to the result of running
-                  <a>parse a smart-poster action record</a> on |ndef|.
-                </li>
-              </ol>
-            </dl>
+        <ol>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`T`" (`0x54`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF text record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`U`" (`0x55`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF URL record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`Sp`" (`0x53` `0x70`)</strong>,
+            set |record| to the result of running
+            <a>parse an NDEF smart-poster record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`s`" (`0x73`)</strong>
+            and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster size record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`t`" (`0x74`)</strong>
+            and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster type record</a> on |ndef|.
+          </li>
+          <li>
+            If |ndef|'s |type:string| is <strong>"`act`" (`0x65` `0x63` `0x74`)
+            </strong> and if |context| is equal to `"smart-poster"`,
+            set |record| to the result of running
+            <a>parse a smart-poster action record</a> on |ndef|.
+          </li>
+          <li>
+            Otherwise, if running the <a>validate local type</a> steps on
```

Comment:
Please remove "Otherwise, " now that we have a final otherwise at the end.

Reply:
Done